gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc.py.j2

```diff
@@ -43,6 +43,49 @@ from google.longrunning import operations_pb2 # type: ignore
 from .base import {{ service.name }}Transport, DEFAULT_CLIENT_INFO
 
 
+
+try:
+    from google.api_core import client_logging
+    CLIENT_LOGGING_SUPPORTED = True
+except ImportError:
+    CLIENT_LOGGING_SUPPORTED = False
+
+import logging
+_LOGGER = logging.getLogger(__name__)
```

Comment:
```suggestion
import logging as std_logging
_LOGGER = std_logging.getLogger(__name__)
```

Reply:
We don't need `std_logging` here because we don't directly import any generated types in this file